apps/fluent-tester/src/RNTester/TestComponents/Callout/CalloutTest.tsx

```diff
@@ -68,22 +87,44 @@ export const CalloutTest: React.FunctionComponent<{}> = () => {
       </Text>
       <Separator />
       <View ref={stdBtnRef} style={{ flexDirection: 'row' }}>
-        <Button content="Press for Callout" onClick={toggleShowStandardCallout} />
-        <Text>
-          <Text>Visibility: </Text>
-          {isStandardCalloutVisible ? <Text style={{ color: 'green' }}>Visible</Text> : <Text style={{ color: 'red' }}>Not Visible</Text>}
-        </Text>
+        <View style={{ flexDirection: 'column' }}>
+          <View style={{ flexDirection: 'row' }}>
+            <Switch value={shouldSetInitialFocus} onValueChange={onInitialFocusChange} />
+            <Text>Set Initial Focus</Text>
+          </View>
+
+          <View style={{ flexDirection: 'row' }}>
+            <Switch value={customRestoreFocus} onValueChange={onRestoreFocusChange} />
+            <Text>Customize Restore Focus</Text>
+          </View>
+        </View>
+
+        <Separator vertical />
+
+        <View style={{ flexDirection: 'column' }}>
+          <Button content="Press for Callout" onClick={toggleShowStandardCallout} />
+          <Text>
+            <Text>Visibility: </Text>
+            {isStandardCalloutVisible ? <Text style={{ color: 'green' }}>Visible</Text> : <Text style={{ color: 'red' }}>Not Visible</Text>}
+          </Text>
+        </View>
       </View>
 
+      <Separator />
+
+      <Button componentRef={decoyBtn1Ref} content="Custom reFocus w/ focus in Callout" />
+      <Button componentRef={decoyBtn2Ref} content="Custom reFocus w/o focus in Callout" />
+
       {showStandardCallout && (
         <Callout
           target={anchorRef}
           onDismiss={onDismissStandardCallout}
           onShow={onShowStandardCallout}
+          onRestoreFocus={customRestoreFocus ? onRestoreFocusStandardCallout : undefined}
           accessibilityLabel="Standard Callout"
-          setInitialFocus
+          setInitialFocus={shouldSetInitialFocus}
         >
-          <View style={{ height: 200, width: 400 }}>
+          <View style={{ padding: 20, borderWidth: 2, borderColor: 'black' }}>
```

Comment:
height and width props no longer necessary?

Reply:
yeah just some padding around the button, looks nicer